docker/Dockerfile

```diff
@@ -58,28 +36,37 @@ RUN conda update -qy conda \
       pip \
       python=3.7
 
-COPY . /app/alphafold
-RUN wget -q -P /app/alphafold/alphafold/common/ \
-  https://git.scicore.unibas.ch/schwede/openstructure/-/raw/7102c63615b64735c4941278d92b554ec94415f8/modules/mol/alg/src/stereo_chemical_props.txt
+COPY . /home/jupyter/alphafold
 
 # Install pip packages.
 RUN pip3 install --upgrade pip \
-    && pip3 install -r /app/alphafold/requirements.txt \
-    && pip3 install --upgrade jax jaxlib==0.1.69+cuda${CUDA/./} -f \
+    && pip3 install -r /home/jupyter/alphafold/requirements.txt \
+    && pip3 install --upgrade jax jaxlib==0.1.69+cuda110 -f \
       https://storage.googleapis.com/jax-releases/jax_releases.html
+   
+# Downgrade because of new base image.
+RUN pip3 install jax==0.2.14 py3Dmol tqdm
+
+# Install alphafold.
+WORKDIR /home/jupyter/alphafold
+RUN python setup.py install
 
 # Apply OpenMM patch.
 WORKDIR /opt/conda/lib/python3.7/site-packages
-RUN patch -p0 < /app/alphafold/docker/openmm.patch
+RUN patch -p0 < /home/jupyter/alphafold/docker/openmm.patch
+
+# Creating a tmp location for jackhmmr; not mounting through to host though.
+RUN sudo mkdir -m 777 --parents /tmp/ramdisk
+
+# Load in the notebook
+COPY --chown=jupyter notebooks/AlphaFold.ipynb /home/jupyter/AlphaFold.ipynb
+COPY notebooks/AlphaFold.ipynb /home/jupyter/AlphaFold_root.ipynb
 
 # We need to run `ldconfig` first to ensure GPUs are visible, due to some quirk
 # with Debian. See https://github.com/NVIDIA/nvidia-docker/issues/1399 for
 # details.
 # ENTRYPOINT does not support easily running multiple commands, so instead we
 # write a shell script to wrap them up.
-WORKDIR /app/alphafold
-RUN echo $'#!/bin/bash\n\
-ldconfig\n\
-python /app/alphafold/run_alphafold.py "$@"' > /app/run_alphafold.sh \
-  && chmod +x /app/run_alphafold.sh
-ENTRYPOINT ["/app/run_alphafold.sh"]
```

Comment:
Entrypoint is inherited from new base image, so that Vertex Workbench can launch